linuxprivchecker.sh

```diff
@@ -170,12 +170,15 @@ filePERMISSIONS(){
   cmdRESPONSE "find / \( -wholename '/home/homedir*' -prune \) -o \( -type d -perm -o+w \) -exec ls -ld '{}' ';' | grep root";
 
   systemNAME="World Writeables Directories for non-root Users";
-  cmdRESPONSE "find / \( -wholename '/home/homedir*' -prune \) -o \( -type d -perm -0002 \) -exec ls -ld '{}' ';' | grep -v root ";
+  cmdRESPONSE "find / \( -wholename '/home/homedir*' -prune \) -o \( -type d -perm -0002 \) -exec ls -ld '{}' ';' | grep -v root";
```

Comment:
hey while you are at it, why is it pruning the weird `/home/homedir*` part? I feel like that dir is completely non standard and personally I would love to know if I can write to another users home dir... This goes for line 176 too.